AlertaDengue/dados/templates/alert_base.html

```diff
@@ -244,7 +244,29 @@ <h4>{% blocktrans %} Últimas {{ look_back }} semanas: {% endblocktrans %}</h4>
 
         <div class="col-md-3 bluebox p-3">
             <h3><span class="glyphicon glyphicon-map-marker" aria-hidden="true"></span> {% blocktrans %}Suspeita de um criadouro?{% endblocktrans %}</h3>
-            <p>{% blocktrans %}Em caso de suspeita de criadouro de mosquitos, ligue para a <a href="http://www.1746.rio.gov.br/" style="color: #FFFFFF">Central de atendimento ao cidadão 1746</a>.{% endblocktrans %}</p>
+            <p>
+            
+            {% blocktrans %}Em caso de suspeita de criadouro de mosquitos, {% endblocktrans %}
+            <!-- RJ -->    
+            {% if state == 'Rio de Janeiro' %}
+                {% blocktrans %}contacte a central de apoio ao cidadão: 1746{% endblocktrans %}
+            <!-- ES -->    
+            {% elif state == 'Espírito Santos' %}
+                {% blocktrans %}contacte a secretaria de saúde de sua cidade{% endblocktrans %}   
+            <!-- PR -->    
+            {% elif state == 'Paraná' %}
+                {% blocktrans %}<a href="http://www.dengue.pr.gov.br/modules/conteudo/conteudo.php?conteudo=19" style="color: #FFFFFF" target="_blank">contacte a secretaria de saúde de sua cidade</a>.{% endblocktrans %}
```

Comment:
I think you can use the blocktrans inside the tag <a> .. because the tag <a>  doesn't need to be inside the translation.

Reply:
agree.

Reply:
ok